### PR TITLE
first components, new dependencies and assets

### PR DESCRIPTION
**Pull request by deniscapp**
- First components of the UI;

- static assets and from packages;

- URL from deployed API.

### CODE REVIEW

**Comment by deniscapp** [comment]
Sorry for too many commits at once! 😕

**Comment by jgcmarins** [comment]
no problem

File: src/Environment.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   // Define a function that fetches the results of an operation (query/mutation/etc)
   // and returns its results as a Promise:
   function fetchQuery(operation, variables, cacheConfig, uploadables) {
-    return fetch('http://localhost:4001', {
+    return fetch('https://chuck-say-api-dwwevjkboq.now.sh/', {
```
Comment (jgcmarins) [inline src/Environment.js:14]: nice one
we can configure an `alias` later on

**Review by jgcmarins — APPROVED** [review]
great job

File: src/Environment.js (hunk 1 above)
Comment (deniscapp) [reply src/Environment.js:14]: For sure

**Pull request merged**